root/static/scripts/release-editor/trackParser.js

```diff
@@ -134,15 +134,16 @@ const trackParser = releaseEditor.trackParser = {
       return data;
     });
 
-    sortByNumber(dataTrackPairs, x => -x.similarity).forEach(function (match) {
-      var data = match.data;
-      var track = match.track;
-
-      if (!data.matchedTrack && !matchedTracks[track.uniqueID]) {
-        data.matchedTrack = track;
-        matchedTracks[track.uniqueID] = 1;
-      }
-    });
+    sortByNumber(dataTrackPairs, x => -x.similarity)
+      .forEach(function (match) {
+        var data = match.data;
+        var track = match.track;
+
+        if (!data.matchedTrack && !matchedTracks[track.uniqueID]) {
+          data.matchedTrack = track;
+          matchedTracks[track.uniqueID] = 1;
+        }
+      });
```

Comment:
To make the diff smaller here I'd revert and just change `forEach(function (match) {` to `forEach((match) => {`